src/spotbugs/spotbugs-excludes.xml

```diff
@@ -252,15 +222,6 @@
         <Bug pattern="NP_NONNULL_FIELD_NOT_INITIALIZED_IN_CONSTRUCTOR"/>
         <Class name="jenkins.util.SystemProperties"/>
```

Comment:
BTW I do not follow why entries like this were ever added to begin with. Simpler to just add `@SuppressFBWarnings` to the class in question with a `justification`?

Reply:
Try removing these entries and adding `@SuppressFBWarnings` annotations for untriaged violations and see how long it takes you.

Reply:
Well, I meant for the entries that have just one or two `<Class>`es.

Reply:
I don't suppose it matters much either way, but I find it easier to triage stuff if the list of stuff to triage is in one place. I did add ~25 `@SuppressFBWarnings(value = "[…]", justification = "TODO needs triage")` annotations to various files where I couldn't easily suppress the warnings from `spotbugs-excludes.xml` (for example, because it was for an anonymous inner class that I couldn't easily refer to), but for the most part I added untriaged stuff to `spotbugs-excludes.xml` to keep the list in one place. I find it convenient to have it organized by violation type and class name in one file. Makes it easy to pick off a section and tackle it one area at a time.